doc/dev/osd_internals/erasure_coding/ecbackend.rst

```diff
@@ -149,23 +149,23 @@ written. The RADOS operation is acknowledged.
 OSD Object Write and Consistency
 --------------------------------
 
-Regardless of the algorithm chosen above, writing of the data is a two
+Regardless of the algorithm chosen above, writing of the data is a two-
 phase process: commit and rollforward. The primary sends the log
 entries with the operation described (see
 osd_types.h:TransactionInfo::(LocalRollForward|LocalRollBack).  
 In all cases, the "commit" is performed in place, possibly leaving some
 information required for a rollback in a write-aside object.  The
 rollforward phase occurs once all acting set replicas have committed
-the commit (sorry, overloaded term) and removes the rollback information.
+the commit (sorry, overloaded term) and remove the rollback information.
```

Comment:
we can remove `(sorry, overloaded term)` here 
```suggestion
the commit, it then removes the rollback information.
```
this is a follow-up operation, the reader might misunderstand not understand it's interdependence with this change